admin/app/controllers/ContentPerformanceController.scala

```diff
@@ -194,4 +197,16 @@ object ContentPerformanceController extends Controller with AuthLogging with Log
         NoCache(Ok(views.html.contentLiveBlog("PROD", socialInteractionsChart, "Live Blog Performance", reportTimestamp)))
     }
   }
+
+  def renderVideoEncodingsDashboard() = AuthActions.AuthActionTest { request =>
+
+    val videoEncodingsReport = jobs.VideoEncodingsJob.getReport("missing-encodings")
+    videoEncodingsReport match {
+      case List() => NoCache(Ok("There are no reported encodings missing as of: %s".format(missingVideoEncodingDateTimeFormat.print(DateTime.now())) ))
+      case List(("Not", "Yet", "Ready")) => NoCache(Ok("Missing video encoding: report has not yet generated"))
```

Comment:
I want to distinguish between the cases where there's nothing to show and the case where the job has yet to run